workflow/workflow_commands.go

```diff
@@ -292,18 +297,23 @@ func printWorkflowProgress(c *cli.Context, wid, rid string, watch bool) error {
 		fmt.Println(color.Magenta(c, "Progress:"))
 	}
 
-	var lastEvent historypb.HistoryEvent // used for print result of this run
+	var lastEvent historypb.HistoryEvent
```

Comment:
Why remove the comment?

Reply:
if `lastEvent` usage changes later in code then the comment needs to be updated as well. Figured it's better to give a good name to the variable than commenting it (the name seems to be already good)

Reply:
changed to wfResult, should clarify the variable better and not need the comment?